src/SankeyPlots.jl

```diff
@@ -121,6 +121,8 @@ In addition to [Plots.jl attributes](http://docs.juliaplots.org/latest/attribute
                             fillcolor := node_colors[mod1(i, end)]
                         elseif edge_color === :dst
                             fillcolor := node_colors[mod1(k, end)]
+                        elseif typeof(edge_color) <: AbstractDict{Tuple{Int64, Int64}, <: Colorant}
+                            fillcolor := edge_color[(i, k)]
```

Comment:
Before doing that, I'd recommend to add a check on whether the (i, k) element is available in the dictionary, and, otherwise use a default value (e.g. :gray).

A warning when that occurs may be useful, but I wouldn't put it inside this loop, otherwise the REPL would be filled up by too many comments.
Such a check could also be performed at the beginning of the script only once

Reply:
Should be resolved by 8183a22426efa29230393597e887a6a63b8e9853